packages/cozy-pouch-link/src/db/pouchdb/pouchdb.js

```diff
@@ -0,0 +1,110 @@
+import { createIndex, getIndexFields, getIndexName } from '../../mango'
+import DatabaseQueryEngine from '../dbInterface'
+import { isMissingPouchDBIndexError } from '../../errors'
+import { getDocsAndNormalize } from './getDocs'
+
+export default class PouchDBQueryEngine extends DatabaseQueryEngine {
+  constructor(pouchManager, doctype) {
+    super()
+    this.pouchManager = pouchManager
+    this.client = pouchManager?.client
```

Comment:
Why is the `?.` needed here? Seems like a typo (same comment in sql engine)

Reply:
It was breaking tests. Not a good reason as we can just better mock, but it does not cost to add it